cdc/owner/changefeed.go

```diff
@@ -970,6 +968,7 @@ func (c *changefeed) handleBarrier(ctx context.Context,
 		barrier.MinTableBarrierTs = barrierTs
 	}
 
+	c.metricsChangefeedBarrierTsGauge.Set(float64(oracle.ExtractPhysical(barrier.MinTableBarrierTs)))
```

Comment:
Please add some comments to explain that MinTableBarrierTs is always the next barrier that blocking the gloabal resolvedTs.